### PR TITLE
doc: Update README with common badges

### PR DESCRIPTION
**Pull request by VersBinarii**

### CODE REVIEW

**Review by szagi3891 — APPROVED** [review]

**Review by sfisol — DISMISSED** [review]

**Comment by sfisol** [comment]
Thank you for contributing!

Please wait until we fix the workflow...

**Comment by sfisol** [comment]
Hi @VersBinarii, can you rebase?

**Comment by VersBinarii** [comment]
Done

File: README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
-# Vertigo - reactive webassembly
+[![Crates.io](https://img.shields.io/crates/d/vertigo.svg)](https://crates.io/crates/vertigo)
+[![Crates.io](https://img.shields.io/crates/v/vertigo.svg)](https://crates.io/crates/vertigo)
+[![Released API docs](https://docs.rs/vertigo/badge.svg)](https://docs.rs/vertigo)
+
+Vertigo - reactive webassembly
```
Comment (sfisol) [inline README.md:5]: `#` deleted after conflict resolving

**Review by sfisol — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (VersBinarii) [reply README.md]: 🤦🏻‍♂️ Fixed

**Review by sfisol — APPROVED** [review]

**Pull request merged**